app/src/parser.c

```diff
@@ -25,14 +25,6 @@
 #include "hdpath.h"
 #include "app_mode.h"
 
-#if defined(TARGET_NANOX)
-// For some reason NanoX requires this function
```

Comment:
No longer required?

Reply:
I tested it and it is not required. I guess this is a remnant from times when nano x sdk was quite new.